examples/llama2/llama2_generate.json

```diff
@@ -89,44 +107,8 @@
                             "priority": 1
                         }
                     ],
+                    "data_config": "transformer_prompt_dummy_data",
```

Comment:
maybe rename this metric to `generation_latency`?

Reply:
Updated